testing/jupyterlab/regression-tests/no-tab-trap-initial-page.test.ts

```diff
@@ -14,6 +14,10 @@ import { test } from "@jupyterlab/galata";
 
 test.describe("when pressing tab key repeatedly", () => {
   test("should cycle through elements back to the start", async ({ page }) => {
+    test.info().annotations.push({
+      type: "link",
+      description: "/no-tab-trap-initial-page.md"
```

Comment:
When looking at the HTML report this "link" displays the bare file name is this intended? or was it intended to be a clickable link to the URL?

Reply:
Yeah I noticed that after the fact. This needs a rethink.

Reply:
let's make an issue to fix this later